content/en/logs/guide/collect-google-cloud-logs-with-push.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Collect Google Cloud Logs with a Pub/Sub Push Subscription
+kind: documentation
+further_reading:
+- link: "/logs/explorer/"
+  tag: "Documentation"
+  text: "Learn how to explore your logs"
+- link: "/logs/explorer/#visualize"
+  tag: "Documentation"
+  text: "Perform Log Analytics"
+- link: "/logs/log_configuration/processors"
+  tag: "Documentation"
+  text: "Learn how to process your logs"
+---
+
+<div class="alert alert-danger">
+This page describes deprecated features with configuration information relevant to legacy Pub/Sub Push subscriptions, useful for troubleshooting or modifying legacy setups. Use a <strong>Pull</strong> subscription with the Datadog Dataflow template to forward your Google Cloud logs to Datadog instead. See <a href="https://docs.datadoghq.com/integrations/google_cloud_platform/#log-collection" target="_blank">Log collection</a> on the Google Cloud integration page for instructions.
+</div>
+
+## Overview
+
+This guide describes how to forward logs from your Google Cloud services to Datadog through a **Push** subscription to a [Google Cloud Pub/Sub][10] topic.
```

Comment:
Might want to use `fix-links` to reorder the the link list.

Reply:
Thanks, done in [184df42](https://github.com/DataDog/documentation/pull/19467/commits/184df42f54e4e08fecf27b3b2c1298e08bbb7bcf)!